src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/time.tsx

```diff
@@ -7,33 +7,65 @@ import Tooltip from 'app/components/tooltip';
 import getDynamicText from 'app/utils/getDynamicText';
 import TextOverflow from 'app/components/textOverflow';
 
-const getBreadcrumbTimeTooltipTitle = (timestamp: string) => {
+const getTooltipTitle = (
+  timestamp: string,
+  relativeTime: string,
+  displayRelativeTime?: boolean
+) => {
   const parsedTimestamp = moment(timestamp);
-  const timestampFormat = parsedTimestamp.milliseconds() ? 'll H:mm:ss.SSS A' : 'lll';
-  return parsedTimestamp.format(timestampFormat);
+  const date = parsedTimestamp.format('ll');
+
+  if (!displayRelativeTime) {
+    const formattedTimestamp = moment(timestamp).format('ll H:mm');
+    const formattedRelativeTime = moment(relativeTime).format('ll H:mm');
```

Comment:
Should we take into account the user's 24 hours preference? There are some existing helper functions in `app/utils/dates`

Reply:
Hm... I believe it's not gonna make any difference here since we are using the `moment.from `method to display the relative time

Reply:
Oh, what about in other parts where we do show the actual time?